docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -2239,6 +2239,23 @@ <h1>Samza Configuration Reference</h1>
                         60 seconds.
                     </td>
                 </tr>
+                <tr>
```

Comment:
What do you think about consolidating metrics-reporting configs into "metric.reporting.level", which can be set to either "info", "debug" or "critical". By default, all metric levels are reported with "info". 

A nice property is that we probably wouldn't require 2 configs: `metrics.timer.debug.enabled` and `metrics.timer.enabled`.. It would also allow you to broaden the scope instead of just timers.

Reply:
This idea is confusing to the users since in the code they don't define the levels.

Reply:
thanks, maybe add it to the config-reference table in the new documentation? 

also, document the relationship between the 2 metrics? eg: the override behavior when metrics.timer.enabled = false and metrics.timer.debug.enabled = true

Reply:
Right, I made the condition to be based on both timer and timer.debug enabled.